ci/jenkins/testkomodo.sh

```diff
@@ -49,9 +49,7 @@ start_tests () {
         --ignore="tests/libres_tests/res/enkf/test_res_config.py"         \
         --ignore="tests/libres_tests/res/enkf/test_site_config.py"        \
         --ignore="tests/libres_tests/res/enkf/test_workflow_list.py"      \
-        --ignore="tests/libres_tests/res/enkf/test_hook_manager.py"       \
-        --ignore="tests/legacy"                              \
-        --ignore="tests/test_formatting.py"
```

Comment:
Should we also delete `tests/test_formatting.py` file? Now it is in `tests/ert_tests` and it is empty

Reply:
It will be deleted if no objections are brought up